tests/search/test_Exchange.py

```diff
@@ -380,10 +380,13 @@ def test_exchange_with_max_duration_constraint(ok_small, op, max_dur, cost):
         route2.append(Node(loc=loc))
     route2.update()
 
-    # Both routes are substantially longer than 5K duration units. So there's
-    # always a max duration violation. Consolidation - either into a single
-    # route, or more into the same route - is typically improving, especially
-    # when the maximum duration violations are significant.
+    # Without maximum duration, route1 has a duration of 5_229 and no time warp
```

Comment:
I think the previous explanation was a bit off here. Consolidating the routes into a single one is actually likely to increase the total max duration violation, however it reduces the distance significantly. We could also consider setting unit distance cost to 0 to make the test only evaluate the violation but to keep changes minimal I only updated the explanation and values.